app/src/main/scala/org/alephium/explorer/api/UnconfirmedTransactionEndpoints.scala

```diff
@@ -32,9 +32,9 @@ trait UnconfirmedTransactionEndpoints extends BaseEndpoint with QueryParams {
       .tag("Transactions")
       .in("unconfirmed-transactions")
 
-  val listUnconfirmedTransactions: BaseEndpoint[Pagination, ArraySeq[TransactionLike]] =
+  val listUnconfirmedTransactions: BaseEndpoint[Pagination, ArraySeq[UnconfirmedTransaction]] =
```

Comment:
I know it makes more sense to have `UnconfirmedTransaction` here, but then tapir is generating a new schema:

```UnconfirmedTransaction1```

which is not nice when front end is auto-generating their type-script files, having this `1` is pretty ugly.

Until I come with a better idea, let's keep `TransactionLike` to avoid creating a new schemas